libs/fgviewer/src/ApiHandler.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#include "ApiHandler.h"
+
+#include <fgviewer/DebugServer.h>
+
+#include <utils/FixedCapacityVector.h>
+#include <utils/Log.h>
+
+#include <CivetServer.h>
+
+namespace filament::fgviewer {
+
+using namespace std::chrono_literals;
+
+namespace {
+
+auto const& kSuccessHeader = DebugServer::kSuccessHeader;
+auto const& kErrorHeader = DebugServer::kErrorHeader;
+
+} // anonymous
+
+static auto const error = [](int line, std::string const& uri) {
```

Comment:
nit: moving this to the anonymous namespace has the same effect as `static`.

Reply:
Fixed